lua/ulib/modules/cfc_timed_punishments.lua

```diff
@@ -0,0 +1,84 @@
+if CLIENT then return end
+
+require "logger"
+local IsValid = IsValid
+
+TimedPunishments = {
+    logger = Logger( "ULX_TimedPunishments" ),
+    Data = include( "cfc_ulx_commands/timed_punishments/server/data.lua" )( logger ),
+    MakeULXCommands = include( "cfc_ulx_commands/timed_punishments/ulx.lua" )( logger ),
```

Comment:
You can do this? 😕

Reply:
Hell yeah brother 😎

`include` will run the script like a function. In this case, I return a function from those files and use them as such.

If you thought it was better, I could assign them to local vars up top and put them in here instead.

Reply:
Gross, but cool...